app/mentor/models.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from economy.models import SuperModel
+from rest_framework import serializers
+
+
+class Mentor(SuperModel):
+
+    TIME_RANGE = [
+        ('1_5', '1 - 5'),
+        ('5_10', '5 - 10'),
+        ('10_15', '10 - 15'),
+        ('15_20', '15 - 20'),
+        ('20_', '20+'),
+    ]
+
+    profile = models.ForeignKey('dashboard.Profile', related_name='mentor', on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, blank=True)
+    email = models.CharField(max_length=255)
+    org = models.CharField(max_length=255)
+    about = models.CharField(max_length=2000)
```

Comment:
Since this charfield is set to be so large, we can just use `models.TextField(blank=True)` here.

Reply:
This makes me think it might be useful to have `about` on `profile` over `Mentor` ... Then again, maybe most of this information would be useful for most `Profile` cases...  The new FTUX will request the user's experience level, so that'll end up being overlap.  We might be able to get away with adding fields not present on `User` to `Profile` 🤔 i.e.: `experience`, `skills_offered`, etc.  I'm not married to one over the other.

Any thoughts on this @owocki ?

Reply:
isn’t the `Profile` your general identity, and the `Mentor` object is specific to you offering to mentor in some capacity (num hours available, skills you have mastery of, availbility for new mentees, etc)